src/pytorch_lightning/strategies/launchers/subprocess_script.py

```diff
@@ -155,7 +155,7 @@ def _hydra_subprocess_cmd(local_rank: int) -> Sequence[str]:
     else:
         command = [sys.executable, "-m", __main__.__spec__.name]
 
-    # extract the hydra configu
```

Comment:
You can now delete this function, and the one above, and then import it from 

`lightning_lite.strategies.launchers.subprocess_script`

Reply:
Oh, so you want that the _hydra_subprocess function should be the default for lightning_lite and be imported inside the PL subprocess script? Right ? 

I can take a look at tests though not sure how to go about it.

Reply:
@awaelchli I have made the changes let me know if you need anything from my end.

Reply:
Yeah, that's great! exactly what we needed. And I added a test.